programmer's guide directory reorg (#1147)
o3de/o3de.org: content/docs/user-guide/programming/remote-console.md

1. After you update the `.cfg` file, deploy the game to mobile devices.

>    For more information, see [Deploy with O3DE Editor](/docs/user-guide/mobile/android/build-deploy#run-the-deployment-tool) and [Creating a Release App](/docs/user-guide/platforms/ios/creating-release-app/).

Reviewer: Even though this topic is in draft, let's get the intended target link corrected, or - in the case of iOS - what one might presume to be the future correct location.
```suggestion
   For more information, see [Android support for O3DE](/docs/user-guide/platforms/android/)<!-- and [iOS support for O3DE](/docs/user-guide/platforms/ios/)-->.
```
